resources/begrep/sikkerDigitalPost/nyinf/api/openapi_spec.yaml

```diff
@@ -1,173 +1,173 @@
                 format: uuid
-                description: Unik ID til meldingen gitt i SBDH - documentIdentification - instanceIdentifier
+                description: Unik ID for forsendelsen. Gitt som del av standard business document header (SBDH) ved sending (<code>standardBusinessDocument.documentIdentification.instanceIdentifier</code>).
               required: true
         responses:
           200:
-            description: liste med statuser  
+            description: Liste med statuser  
             content:
               application/json:
                 schema:
                   $ref: '#/components/schemas/statuses'          
           401:
-            description: Utløpt token
+            description: Autentiseringsfeil. F.eks. manglande JWT bearer grant.
             content:
               application/problem+json:
                 schema: 
                   $ref : '#/components/schemas/problem'
           403:
-            description: Token mangler "digitalpostinnbygger:send" scope, eller avsender har ikke gyldig avtale med aksesspunktleverandør. 
+            description: Autorisasjonsfeil. F.eks. gyldig JWT bearer grant som mangler påkrevd scope.
             content:
               application/problem+json:
                 schema: 
                   $ref : '#/components/schemas/problem'
     /{version}/messages:
       get: 
-        summary: Henter ned liste med forettningsmeldinger på mottatte forsendelser
+        summary: Hent innkommende forsendelser
+        description: NB. De eneste innkommende forsendelsene som er støttet i versjon 1.0.0 er kvitteringer.
         parameters:
           - in: path
             name: version            
             schema:
               type: string                               
-              description: Versjon av API
+              description: Versjon av APIet
             required: true
           - in: query
             name: mottakeridentifikator            
             schema:
               type: string
-              description: Identifikator til gitt i avsenderidentifikator ved sending av melding
+              description: Unik ID for fagsystem hos virksomhet. Brukes for å støtte at samme virksomhet kan ha flere fagsystem. Gitt som del av forretningsmelding ved sending (<code>digital.avsender.avsenderidentifikator</code>).
           - in: query
             name: page_size            
             schema:
               type: string
-              description: 'Identifikator til gitt i avsenderidentifikator ved sending av melding. TODO: Usikker på om vi trenger paging'
+              description: Antall forsendelser per side.
           - in: query
             name: page            
             schema:
               type: string
-              description: 'Identifikator til gitt i avsenderidentifikator ved sending av melding. TODO: Usikker på om vi trenger paging '         
+              description: Sidenummer for side som skal hentes.
```

Comment:
Skulle paginering utgå i første versjon?

Reply:
Ja, men vi må ha muligheit til å spesifisere antall element som hentes ut.

Tenkte kanskje dere berre kan ignorere page eller feile dersom page er oppgitt - så ligg alt til rette.

Høyres ok ut ?

Reply:
De er optional slik de er definert nå, så klienten trenger ikke angi dem.  Men om de angis fra klient og server ikke støtter dem bør det i allefall beskrives hva som skal skje.  Kan vi returnere er http error eller bør vi bruke en response header som forteller at paging ikke er supported?

Reply:
HTTP 400 m kort beskrivelse i standard JSON slik som andre valideringsfeil for endepunkta?

Så kan eg legge inn ein kommentar i specen.